.changeset/yellow-suns-clean.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/auth': patch
+---
+
+Prevent React Native logging a warning about deprecation of `AsyncStorage` if user has provided the non-deprecated version.
```

Comment:
Is "user" definitely the right term?  Or could it be "the developer" or even "the package.json file" or wherever this is defined?

I clearly do not know :)  But if we go with this, I'd suggest "if the user"

Reply:
"developer" makes sense.